libs/logic-apps-shared/src/utils/src/lib/models/template.ts

```diff
@@ -8,9 +8,17 @@ export interface Manifest {
   skus: SkuType[];
   kinds?: WorkflowKindType[];
   details: Record<string, string>;
+
+  /* This is a markdown to show features for multi-workflow and details in case of single workflow */
   detailsDescription?: string;
+
   tags?: string[];
   artifacts: Artifact[];
+
+  /* This consists of list of workflows listed in the multi-workflow template.
+  The key is the folder name, followed by metadata where name is default name to be used for creation */
+  workflows?: Record<string, { name: string }>;
+
   images: Record<string, string>;
   prerequisites?: string;
   parameters: Parameter[];
```

Comment:
From our last discussion, we decided that parameters/connections will not be present for multi-workflow manifest. Should we consider making these optional fields?

Reply:
I am still not sure we should mark that as optional because again these are app level artifacts, over time when we do implementation for multi workflow we can change this, but can keep it open for now.